spec/_attachments/ic.did

```diff
@@ -36,6 +36,10 @@ type change_details = variant {
     mode : variant {install; reinstall; upgrade};
     module_hash : blob;
   };
+  load_snapshot : record {
+    module_hash : blob;
```

Comment:
We discussed the information stored in load_snapshot in the Interface spec meeting and concluded that the module hash need not be stored since it can be derived from the canister history if we additionally store the canister version of the canister when its snapshot was taken. So our suggestion is to replace the module hash here with the canister version so that a load snapshot record would contain the canister version and timestamp when the snapshot was taken.